pom.xml

```diff
@@ -176,6 +176,17 @@
             <version>${powermock.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>mailer</artifactId>
+            <version>1.8</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sonyericsson.hudson.plugins.rebuild</groupId>
```

Comment:
This is used to send the mentions to the user who performed the rebuild using the rebuild plugin. This is not necessary, so I set the optional property to true.